packages/pf4-component-mapper/src/form-fields/dual-list-select.scss

```diff
@@ -0,0 +1,18 @@
+.ddorg__pf4-component-mapper__dual-list-select-toolbar {
+    height: auto !important;
+    padding: 0px;
+}
+
+.ddorg__pf4-component-mapper__dual-list-select-toolbar-content {
+    padding: 0px 0px 16px 0px !important;
```

Comment:
you can us pf modifier classes like `pf-p-m-0` `pf-u-pr-md` etc. So we can remove the !important nonsense :smile:

Reply:
let's them make the nonsense :see_no_evil: